content/_data/upgrades/2-277-1.adoc

```diff
@@ -0,0 +1,51 @@
+==== Configuration Form Modernization
+
+The transition from HTML `table` layout to HTML `div` layout in configuration forms requires changes in some plugins for compatibility.
+Jenkins administrators should update the installed plugins when they upgrade to Jenkins 2.277.1.
+Some older plugin releases will cause the Jenkins "Configure System" page to fail to save.
+Other older plugin releases will report Java runtime exceptions until they are updated to the most recent release.
+
+Use the Plugin Manager to update plugins immediately after upgrading to Jenkins 2.277.1.
+Make sure to _Check for Updates_ to ensure no outdated cached data hides updates.
+
```

Comment:
not needed? the fix to reload on upgrade / downgrade was backported
```suggestion
```

Reply:
Thanks, I was unaware of that.

The first sentence is still useful though.

```suggestion
Use the Plugin Manager to update plugins immediately after upgrading to Jenkins 2.277.1.
```

Reply:
Removed in 60354e7af801f725f6371d4f217c2b4c300b2b76